### DISCUSSION TITLE
IsADirectoryError

### DISCUSSION THREAD

**Discussion by jic198** [deepmodeling/dpgen#1121] [Q&A]
I am having this strange error and I don't know how to fix it.
`IsADirectoryError: [Errno 21] Is a directory: 'iter.000000/01.model_devi/confs/000.0000.poscar'` The error happed in the beginning of model_devi. I checked my `param.json` and everything looked fine. Do you know the things that could lead to this error? Thank you!

**Comment by njzjz** [comment]
It looks like the same error as #314. I suggest you read #314 and share more details.

**Reply by jic198** [reply]
Just found out the culprit by digging into the code. `sys_configs` should be a list of lists. If you use a list of strings, you are screwed. I hope you can add this info in the documentation so that future users will be aware of it.

**Reply by AroundPeking** [reply]
I have encountered the same issue. And my sys_configs seems right. How can I do?
![1686133451932](https://github.com/deepmodeling/dpgen/assets/108850865/fe2f7f2c-27cd-450f-8666-39cbb9988892)

**Reply by AnguseZhang** [reply]
What's the format of your configuration files? It seems like the format of ABACUS. You need to find out what's the key to load this file in `dpdata` (probably `abacus/stru`), and pass it into "sys_format" in `param.json`. 

See https://docs.deepmodeling.com/projects/dpgen/en/latest/run/param.html#argument:run_jdata/sys_format. 
https://github.com/deepmodeling/dpdata

**Reply by AroundPeking** [reply]
yep, that's how I did it. 
![1686150722976](https://github.com/deepmodeling/dpgen/assets/108850865/94165631-0e92-4df8-a039-e9bfa6fe2f0a)
![1686150775335](https://github.com/deepmodeling/dpgen/assets/108850865/a08afa5a-2c81-4e66-8974-65c624a3e598)
